homeassistant/components/meater/sensor.py

```diff
@@ -189,7 +190,7 @@ def __init__(
             },
             manufacturer="Apption Labs",
             model="Meater Probe",
-            name=f"Meater Probe {device_id}",
+            name=f"Meater Probe {device_id[:8]}",
```

Comment:
Do we know it's safe to truncate the device ID like this? If we end up with several sensors with the same entity_id (but suffixed by a number) it's a huge mess for users to untangle.